django_mongodb/functions.py

```diff
@@ -17,5 +34,31 @@ def extract(self, compiler, connection):
     return {operator: lhs_mql}
 
 
+def func(self, compiler, connection):
+    lhs_mql = process_lhs(self, compiler, connection)
+    try:
+        operator = MONGO_OPERATORS[self.__class__]
```

Comment:
I think it can be done with get: 
`operator = MONGO_OPERATORS.get(self.__class__, self.function.lower())`